docs/plugin_matrix.rst

```diff
@@ -66,6 +66,7 @@ tga                 - star.plu.cn        Yes   No
                     - star.tga.plu.cn
 tv4play             - tv4play.se         Yes   Yes   Streams may be geo-restricted to Sweden.
                                                      Only non-premium streams currently supported.
+tvcatchup           - tvcatchup.com      Yes   No    Streams may be geo-restricted to Great Britain.
                     - fotbollskanalen.se
```

Comment:
You are breaking up the tv4play entry here, the fotobollskanalen.se line should be directly under the tv4play lines.